sweepbatcher/sweep_batcher_test.go

```diff
@@ -94,6 +119,7 @@ func TestSweepBatcherBatchCreation(t *testing.T) {
 			AmountRequested: 111,
 		},
 
+		DestAddr:    destAddr,
```

Comment:
Is `DestAddr` always stored in the DB or only if there's one provided by the client?

Reply:
If I don't set it in tests, `CreateLoopOut` fails in SQL version of the store, inside function [`loopOutToInsertArgs`](https://github.com/lightninglabs/loop/blob/v0.28.5-beta/loopdb/sql_store.go#L491), which attempts to convert the address to string:

```go
DestAddress:         loopOut.DestAddr.String(),
```

This code assumes that it is always filled. I guess, if the client doesn't fill the field manually when creating a swap, it is filled by the `loopd` software before the swap is written to DB.